<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MRuntimeStylesheetEditor</class>
 <widget class="QWidget" name="MRuntimeStylesheetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QPushButton" name="search">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="saveToFile">
       <property name="text">
        <string>&amp;Save to file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="readAppStyle">
       <property name="text">
        <string>&amp;Read style</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="browseFile">
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="loadFromFile">
       <property name="text">
        <string>&amp;Load from file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QFrame" name="styleTextFrame">
        <layout class="QHBoxLayout" name="styleTextFrameLayout"/>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QPushButton" name="applyStyle">
       <property name="text">
        <string>&amp;Apply style</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="autoApplyStyle">
       <property name="text">
        <string>Auto apply style</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="protectStyleEditor">
       <property name="text">
        <string>Protect style editor</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="filenameLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
